velox/functions/prestosql/aggregates/tests/ApproxDistinctTest.cpp

```diff
@@ -236,6 +236,130 @@ TEST_F(ApproxDistinctTest, globalAggStrings) {
   testGlobalAgg(values, kFruits.size());
 }
 
+TEST_F(ApproxDistinctTest, globalAggTimeStamp) {
```

Comment:
I wonder if there is a way to simplify the test. The fix seems rather generic, hence, I hope the test can be simplified.

Reply:
For example, 

```
TEST_F(ApproxDistinctTest, globalAggTimeStamp) {
  auto data = makeFlatVector<Timestamp>(
      1'000, [](auto row) { return Timestamp::fromMillis(row); });
  testGlobalAgg(data, 1010);
}
```

Without the fix this test fails:

```
[ RUN      ] ApproxDistinctTest.globalAggTimeStamp
fbcode/velox/exec/tests/utils/QueryAssertions.cpp:1104: Failure
Failed
Expected 1, got 1
1 extra rows, 1 missing rows
1 of extra rows:
	1006

1 of missing rows:
	1010
```

In Presto,

```
presto> select approx_distinct(date_add('millisecond', x, from_unixtime(0))) from unnest(sequence(0, 999)) as t(x);
 _col0
-------
  1010
(1 row)
```

Reply:
Thanks, updated